grails-app/views/projectReview/_index.gsp

```diff
@@ -4,6 +4,7 @@
         projectKey:'${issue.projectKey}',
         projectUrl: "${createLink(controller: 'project', action: 'getProject')}",
         roles: '${session.roles}'.replace(/\s+/g, '').replace("[", "").replace("]", "").split(","),
+        isAdmin: "${session.isAdmin}"
```

Comment:
See previous comment

Reply:
Even better ... I think if we had a new endpoint that retrieved the user along with their roles/permissions - that would be ideal. That way, you won't have to pass anything in - just query for it.